### PR TITLE
MSDKUI-1568: [Android] Align component names

### PR DESCRIPTION
**Pull request by imvjk**
Aligning android component names equivalent to iOS component name. This changes include changes of following classes --
GuidanceManeuverPanel -> GuidanceManeuverView
GuidanceCurrentSpeedPanel -> GuidanceSpeedView (incl. Presenter, Data)
GuidanceSpeedLimitPanel -> GuidanceSpeedLimitView (incl. Presenter, Data)
GuidanceNextManeuverPanel -> GuidanceNextManeuverView (incl. Presenter, Data)
& ManeuverDescriptionItem -> ManeuverItemView

### CODE REVIEW

File: MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceNextManeuverViewFragmentTest.kt [hunk 1]
```diff
@@ -33,29 +33,29 @@ import org.mockito.Mockito.verify
...
 
 /**
- * Tests for [GuidanceNextManeuverPanelFragment]
+ * Tests for [GuidanceNextManeuverFragment]
  */
-class GuidanceNextManeuverPanelFragmentTest : BaseTest() {
+class GuidanceNextManeuverViewFragmentTest : BaseTest() {
 
-    private lateinit var guidanceNextManeuverPanelFragment: GuidanceNextManeuverPanelFragment
+    private lateinit var guidanceNextManeuverPanelFragment: GuidanceNextManeuverFragment
```
Comment (Docik) [inline MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceNextManeuverViewFragmentTest.kt]: Please consider changing variable name

File: MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelViewFragmentTest.kt [hunk 2]
```diff
@@ -30,23 +30,23 @@ import org.mockito.Mockito.mock
...
 
 /**
- * Tests for [GuidanceCurrentStreetFragment].
+ * Tests for [GuidanceStreetLabelFragment].
  */
-class GuidanceCurrentStreetFragmentTest : BaseTest() {
+class GuidanceStreetLabelViewFragmentTest : BaseTest() {
 
-    lateinit var guidanceCurrentStreetFragment: GuidanceCurrentStreetFragment
+    private lateinit var guidanceCurrentStreetFragment: GuidanceStreetLabelFragment
```
Comment (Docik) [inline MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelViewFragmentTest.kt]: Please consider changing variable name

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceManeuverPresenterTest.java [hunk 3]
```diff
@@ -41,25 +41,25 @@
...
 /**
- * Tests for {@link GuidanceManeuverPanelPresenter}.
+ * Tests for {@link GuidanceManeuverPresenter}.
  */
 @PrepareForTest({ NavigationManager.class, Signpost.class, Image.class })
-public class GuidanceManeuverPanelPresenterTest extends RobolectricTest {
+public class GuidanceManeuverPresenterTest extends RobolectricTest {
 
-    private GuidanceManeuverPanelPresenter mManeuverPanelPresenter;
+    private GuidanceManeuverPresenter mManeuverPanelPresenter;
```
Comment (Docik) [inline MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceManeuverPresenterTest.java]: Please consider changing variable name

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceNextManeuverPresenterTest.java [hunk 4]
```diff
@@ -33,25 +33,25 @@
...
 /**
- * Tests for  {@link GuidanceNextManeuverPanelPresenter}.
+ * Tests for  {@link GuidanceNextManeuverPresenter}.
  */
 @PrepareForTest({ NavigationManager.class })
-public class GuidanceNextManeuverPanelPresenterTest extends RobolectricTest {
+public class GuidanceNextManeuverPresenterTest extends RobolectricTest {
 
-    private GuidanceNextManeuverPanelPresenter mNextManeuverPanelPresenter;
+    private GuidanceNextManeuverPresenter mNextManeuverPanelPresenter;
```
Comment (Docik) [inline MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceNextManeuverPresenterTest.java]: Please consider changing variable name

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedLimitViewTest.java [hunk 5]
```diff
@@ -33,19 +33,19 @@
...
+ * Tests for class {@link GuidanceSpeedLimitView}.
  */
-public class GuidanceSpeedLimitPanelTest extends RobolectricTest {
+public class GuidanceSpeedLimitViewTest extends RobolectricTest {
 
     private static final int VELOCITY = 60;
     private static final int SPEED_LIMIT = 50;
 
-    private GuidanceSpeedLimitPanel mSpeedLimitPanel = null;
+    private GuidanceSpeedLimitView mSpeedLimitPanel = null;
```
Comment (Docik) [inline MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedLimitViewTest.java]: Please consider changing variable name

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedViewTest.java [hunk 6]
```diff
@@ -33,21 +33,21 @@
...
-public class GuidanceCurrentSpeedPanelTest extends RobolectricTest {
+public class GuidanceSpeedViewTest extends RobolectricTest {
 
     private static final int VELOCITY = 60;
     private static final int SPEED_LIMIT = 50;
     private static final int RED_COLOR = 16711680;
     private static final int BLUE_COLOR = 255;
 
-    private GuidanceCurrentSpeedPanel mCurrentSpeedPanel = null;
+    private GuidanceSpeedView mCurrentSpeedPanel = null;
```
Comment (Docik) [inline MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedViewTest.java]: Please consider changing variable name

File: MSDKUIDemo/MSDKUIApp/src/main/res/values-en-rUS/strings.xml [hunk 7]
```diff
@@ -19,7 +19,7 @@
...
   <string name="msdkui_app_about_info_part_one">Build beautiful interfaces for Android and iOS with the HERE Mobile SDK UI Kit</string>
-  <string name="msdkui_app_about_info_part_three">Using just a few lines of code, you can create a full UI with the Premium edition of the HERE Mobile SDK. So instead of needing several hours to build a manoeuvre list using native APIs, you\'ll only need a couple minutes. Less development time means faster time to market. \n\nOf course you can use the UI Kit to change the appearance (colours, dimensions) and behaviour (data presentation, logic, flow, subcomponents) of some UI controls. It\'s all up to you.</string>
+  <string name="msdkui_app_about_info_part_three">Using just a few lines of code, you can create a full UI with the Premium edition of the HERE Mobile SDK. So instead of needing several hours to build a manoeuvre list using native APIs, you\'ll only need a couple minutes. Less development time means faster time to market. \n\nOf course you can use the UI Kit to change the appearance (colours, dimensions) and behaviour (labelData presentation, logic, flow, subcomponents) of some UI controls. It\'s all up to you.</string>
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/res/values-en-rUS/strings.xml]: done

File: MSDKUIDemo/MSDKUIApp/src/main/res/values-tl/strings.xml [hunk 8]
```diff
@@ -19,7 +19,7 @@
...
+  <string name="msdkui_app_about_info_part_three">Sa paggamit lamang ng iilang linya ng code, makakagawa ka na ng buong UI gamit ang Premium edition ng HERE Mobile SDK. Kaya, sa halip na kailangan pa ng ilang araw para makagawa ng listahan ng maneobra gamit ang native APIs, ang kailangan mo lamang ay ilang oras. Mas kaunting panahon sa pag-develop mas mabilis na panahon para i-market.\n\nSiyempre, kung gusto mo, maaari mong gamitin ang UI Kit upang baguhin ang hitsura (mga kulay, dimension) at behaviour (labelData presentation, logic, flow, mga subcomponent) ng ilang kontrol ng UI. Ikaw ang bahala.</string>
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/res/values-tl/strings.xml]: done

File: MSDKUIDemo/MSDKUIApp/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -19,7 +19,7 @@
...
   <string name="msdkui_app_about_info_part_one">Build beautiful interfaces for Android and iOS with the HERE Mobile SDK UI Kit</string>
-  <string name="msdkui_app_about_info_part_three">Using just a few lines of code, you can create a full UI with the Premium edition of the HERE Mobile SDK. So instead of needing several hours to build a manoeuvre list using native APIs, you\'ll only need a couple minutes. Less development time means faster time to market. \n\nOf course you can use the UI Kit to change the appearance (colours, dimensions) and behaviour (data presentation, logic, flow, subcomponents) of some UI controls. It\'s all up to you.</string>
+  <string name="msdkui_app_about_info_part_three">Using just a few lines of code, you can create a full UI with the Premium edition of the HERE Mobile SDK. So instead of needing several hours to build a manoeuvre list using native APIs, you\'ll only need a couple minutes. Less development time means faster time to market. \n\nOf course you can use the UI Kit to change the appearance (colours, dimensions) and behaviour (labelData presentation, logic, flow, subcomponents) of some UI controls. It\'s all up to you.</string>
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/res/values/strings.xml]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceManeuverFragment.kt [hunk 10]
```diff
@@ -54,47 +54,47 @@ class GuidanceManeuverPanelFragment : Fragment(), GuidanceManeuverPanelListener
...
+        fun newInstance() = GuidanceManeuverFragment()
     }
 
     /**
      * Creates Panel View.
      */
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View? {
-        val panelFragment = GuidanceManeuverPanel(activity)
+        val panelFragment = GuidanceManeuverView(activity)
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceManeuverFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceManeuverFragment.kt [hunk 11]
```diff
@@ -54,47 +54,47 @@ class GuidanceManeuverPanelFragment : Fragment(), GuidanceManeuverPanelListener
...
         panelFragment.id = R.id.guidanceManeuverPanelId
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceManeuverFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceNextManeuverFragment.kt [hunk 12]
```diff
@@ -34,51 +34,51 @@ import kotlinx.android.extensions.ContainerOptions
...
-        fun newInstance() = GuidanceNextManeuverPanelFragment()
+        fun newInstance() = GuidanceNextManeuverFragment()
     }
 
     /**
      * Creates Panel View.
      */
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        val panelFragment = GuidanceNextManeuverPanel(activity)
+        val panelFragment = GuidanceNextManeuverView(activity)
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceNextManeuverFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceNextManeuverFragment.kt [hunk 13]
```diff
@@ -34,51 +34,51 @@ import kotlinx.android.extensions.ContainerOptions
...
         panelFragment.id = R.id.guidanceNextManeuverPanelId;
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceNextManeuverFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceSpeedLimitFragment.kt [hunk 14]
```diff
@@ -50,13 +50,13 @@ class GuidanceSpeedLimitFragment : Fragment(), GuidanceSpeedListener  {
      */
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View? {
-        val panelFragment = GuidanceSpeedLimitPanel(activity)
+        val panelFragment = GuidanceSpeedLimitView(activity)
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceSpeedLimitFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceSpeedLimitFragment.kt [hunk 15]
```diff
@@ -50,13 +50,13 @@ class GuidanceSpeedLimitFragment : Fragment(), GuidanceSpeedListener  {
...
         panelFragment.id = R.id.guidanceSpeedLimitPanelId
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceSpeedLimitFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelFragment.kt [hunk 16]
```diff
@@ -22,73 +22,64 @@ import android.view.LayoutInflater
...
+        fun newInstance() = GuidanceStreetLabelFragment()
     }
 
     /**
      * Creates Panel View.
      */
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                               savedInstanceState: Bundle?): View? {
-        val panelFragment = GuidanceCurrentStreet(activity)
+        val panelFragment = GuidanceStreetLabelView(activity)
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelFragment.kt [hunk 17]
```diff
@@ -22,73 +22,64 @@ import android.view.LayoutInflater
...
         panelFragment.id = R.id.guidanceCurrentStreetId
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelFragment.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_route_preview.xml [hunk 18]
```diff
@@ -68,7 +68,7 @@
         app:layout_constraintBottom_toTopOf="@+id/see_steps"
         app:layout_constraintTop_toBottomOf="@+id/divider" />
 
-    <com.here.msdkui.routing.ManeuverDescriptionList
+    <com.here.msdkui.routing.ManeuverList
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/main/res/layout/guidance_route_preview.xml]: done

File: MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceManeuverViewFragmentTest.kt [hunk 19]
```diff
@@ -32,27 +32,27 @@ import org.mockito.Mockito.verify
...
-    private lateinit var mockPanelPresenter: GuidanceManeuverPanelPresenter
+    private lateinit var mockPresenter: GuidanceManeuverPresenter
 
     @Before
     fun setup() {
         super.setUp()
         MockitoAnnotations.initMocks(this)
         navigationManager = mock(NavigationManager::class.java)
-        guidanceManeuverPanelFragment = GuidanceManeuverPanelFragment.newInstance()
+        guidanceManeuverPanelFragment = GuidanceManeuverFragment.newInstance()
```
[the remark this answers is not in the document]
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceManeuverViewFragmentTest.kt]: done

File: MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelViewFragmentTest.kt (hunk 2 above)
Comment (imvjk) [reply MSDKUIDemo/MSDKUIApp/src/test/java/com/here/msdkuiapp/guidance/GuidanceStreetLabelViewFragmentTest.kt]: done

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceManeuverPresenterTest.java (hunk 3 above)
Comment (imvjk) [reply MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceManeuverPresenterTest.java]: done

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceNextManeuverPresenterTest.java (hunk 4 above)
Comment (imvjk) [reply MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceNextManeuverPresenterTest.java]: done

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedViewTest.java (hunk 6 above)
Comment (imvjk) [reply MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedViewTest.java]: done

File: MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedLimitViewTest.java (hunk 5 above)
Comment (imvjk) [reply MSDKUIKit/MSDKUILib/src/test/java/com/here/msdkui/guidance/GuidanceSpeedLimitViewTest.java]: done

**Pull request merged**